pkgs/top-level/all-packages.nix

```diff
@@ -10167,7 +10167,10 @@ in
 
   fontconfig-ultimate = callPackage ../development/libraries/fontconfig-ultimate {};
 
-  folly = callPackage ../development/libraries/folly { };
+  folly = callPackage ../development/libraries/folly {
+    # Clang 5 miscompiles folly. Build with a newer compiler on Darwin.
+    stdenv = if stdenv.isDarwin then llvmPackages_7.stdenv else stdenv;
```

Comment:
Not needed anymore since ae1e940cc0c4767adba1ea8f4a25bda569be26c5 has just landed on master #60491